Tests/DefaultXpringClientTest.swift

```diff
@@ -265,6 +266,88 @@ final class DefaultXpringClientTest: XCTestCase {
     XCTAssertThrowsError(try xpringClient.getTransactionStatus(for: .testTransactionHash))
   }
 
+  // MARK: - Account Existence
+
+  func testAccountExistsWithSuccess() {
+    // GIVEN a XpringClient which will successfully return a balance from a mocked network call.
+    let xpringClient = DefaultXpringClient(networkClient: FakeNetworkClient.successfulFakeNetworkClient)
+
+    // WHEN the existence of the account is checked.
+    guard let exists = try? xpringClient.accountExists(for: .testAddress) else {
+      XCTFail("Exception should not be thrown when checking existence of valid account")
+      return
+    }
+
+    // THEN the balance is correct.
+    XCTAssertEqual(exists, true)
```

Comment:
You can use `XCTAssertTrue` or `XCTAssertFalse` (here and elsewhere)

Reply:
done